frontend/src/pages/user-profile/components/user-profile-sidebar/user-profile-sidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import { useAppSelector } from '#libs/hooks/hooks.js';
+import { type UserAuthResponseDto } from '#packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+const UserProfileSidebar: React.FC = () => {
+  const { authenticatedUser } = useAppSelector(({ auth }) => ({
+    authenticatedUser: auth.authenticatedUser,
+  }));
+
+  return (
+    <div className={styles['container']}>
+      <div className={styles['header']}>Profile</div>
+      <div className={styles['user-info']}>
+        <div className={styles['avatar-container']}>
+          <img
+            src="images/avatar.png"
```

Comment:
First import that component and reuse it here

Reply:
done